install_config/cluster_metrics.adoc

```diff
@@ -329,6 +329,11 @@ appended to the prefix starting from 1.
 |`openshift_metrics_cassandra_pvc_size`
 |The persistent volume claim size for each of the Cassandra nodes.
 
+|`openshift_metrics_cassandra_storage_class_name`
+|The specified storage class for the Cassandra persistent volume claim. The
```

Comment:
I might say, "If you want to explicitly set the storage class, you must not set \`openshift_metrics_cassandra_storage_type\` to  \`emptydir\` or \`dynamic`"

Do we explain what else can you set it to somewhere that a user will find?

Reply:
Thanks! I'll push an update in a few minutes. We do actually have quite a lot of content around the setting the storage class, so I think it's good. :cat:

Reply:
I figured that it was already covered. :) Thanks!